client/lxd.go

```diff
@@ -382,11 +382,16 @@ func (r *ProtocolLXD) queryStruct(method string, path string, data any, ETag str
 
 // queryOperation sends a query to the LXD server and then converts the response metadata into an Operation object.
 // It sets up an early event listener, performs the query, processes the response, and manages the lifecycle of the event listener.
-func (r *ProtocolLXD) queryOperation(method string, path string, data any, ETag string) (Operation, string, error) {
+func (r *ProtocolLXD) queryOperation(method string, path string, data any, ETag string, skipListener bool) (Operation, string, error) {
```

Comment:
Can we call the new arg something like `useEventListener` rather than `skip...` so the logic isnt inverted.

Reply:
I've renamed the argument to `useEventListener`. The field on `operation` is still `skipListener` though, so that the default value continues to use the events API.